rbi/stdlib/uri.rbi

```diff
@@ -21,7 +21,290 @@
+# :       Executes the given command and returns its pid without waiting for
+#         completion.
+#
+#     [`system`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-system)
+# :       Executes the given command in a subshell.
+#
+#
+#
+# ### Loading
+#
+#     [`autoload`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-autoload)
+# :       Registers the given file to be loaded when the given constant is first
+#         referenced.
+#
+#     [`load`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-load)
+# :       Loads the given Ruby file.
+#
+#     [`require`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-require)
+# :       Loads the given Ruby file unless it has already been loaded.
+#
+#     [`require_relative`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-require_relative)
+# :       Loads the Ruby file path relative to the calling file, unless it has
+#         already been loaded.
+#
+#
+#
+# ### Yielding
+#
+#     [`tap`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-tap)
+# :       Yields `self` to the given block; returns `self`.
+#
+#     [`then`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-then) (aliased as [`yield_self`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-yield_self))
+# :       Yields `self` to the block and returns the result of the block.
+#
+#
+#
+# ### [`Random`](https://docs.ruby-lang.org/en/2.7.0/Random.html) Values
+#
+#     [`rand`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-rand)
+# :       Returns a pseudo-random floating point number strictly between 0.0 and
+#         1.0.
+#
+#     [`srand`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-srand)
+# :       Seeds the pseudo-random number generator with the given number.
+#
+#
+#
+# ### Other
+#
+#     [`eval`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-eval)
+# :       Evaluates the given string as Ruby code.
+#
+#     [`loop`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-loop)
+# :       Repeatedly executes the given block.
+#
+#     [`sleep`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-sleep)
+# :       Suspends the current thread for the given number of seconds.
+#
+#     [`sprintf`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-sprintf) (aliased as [`format`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-format))
+# :       Returns the string resulting from applying the given format string to
+#         any additional arguments.
+#
+#     [`syscall`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-syscall)
+# :       Runs an operating system call.
+#
+#     [`trap`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-trap)
+# :       Specifies the handling of system signals.
+#
+#     [`warn`](https://docs.ruby-lang.org/en/2.7.0/Kernel.html#method-i-warn)
+# :       Issue a warning based on the given messages and options.
```

Comment:
I was curious why this comment is showing up here, and I put up a PR to make it go away: #6285